build.js

```diff
@@ -184,6 +184,24 @@ autobase.register({
   ]
 })
 
+autobase.register({
+  name: 'trace',
+  compact: true,
+  fields: [
+    {
```

Comment:
do we need to declare this encoding? the view index can be implied by the offset in the array if we ensure ordering when we write

it would have the trade off that we have to write empty arrays for views that weren't touched

Reply:
true indeed, let's leave as it is then